### PR TITLE
Implement HTML as an output option

### PR DESCRIPTION
**Pull request by cuducos**
**Disclaimer**: This PR depends on #5 — its _diff_ includes #5's _diff_ and merging it will automatically merge #5 as well. It might be a good idea to review #5 before this one.

This implements the HTML output option using the `showdown` package to convert the existing Markdown to HTML. Now the CLI accepts `--output html` and the `engine` function accepts `"html"` as its second argument (which stands for `format` in the source code).

This new option is automatically added to the CLI's help:

```console
$ jsv --help
Usage: jsv [options] [<json>]

jsv (JSON Scheme Viewer)

JSON Schema viewer is a lightweight javascript library and tool that turns JSON
schemas into elegant human-readable documents.

It expects a JSON Schema from stdin and outputs to stdout its version for
visualization in MarkDown, unless another format is passed using --output.

Options:
  -V, --version          output the version number
  -p, --output <format>  Format of the output: html, md (default: "md")
  -h, --help             display help for command
```

And it can be testes with this simple example (included in the `README.md`):

```console
$ jsv --output html '{"$schema": "http://json-schema.org/draft-04/schema#", "title": "Data Resource", "description": "Data Resource.", "type": "object"}'
<h1 id="dataresource">Data Resource</h1>
<p><strong>(<code>object</code>)</strong></p>
<p>Data Resource.</p>
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,47 +10,99 @@ Requires [NodeJS](https://nodejs.org/en/) 12+:
...
 
 jsv (JSON Scheme Viewer)
 
 JSON Schema viewer is a lightweight javascript library and tool that turns JSON
 schemas into a elegant human readable documents.
 
-It expects URL to be a location of a JSON scheme file, and it decides the
-format of the output based on the file extension of OUTPUT (which is created
-as a result) or the format provided via the --output option.
+It expects a JSON Schema from stdin and outputs to stdout its version for
```
Comment (rufuspollock) [inline README.md:30]: I don't line wrap at 80 characters in READMEs usually.

File: README.md [hunk 2]
```diff
@@ -10,47 +10,99 @@ Requires [NodeJS](https://nodejs.org/en/) 12+:
...
 
 Options:
-  -V, --version      output the version number
-  --output=<output>  Format of the output file.
-  --css=<css>        URL of a CSS stylesheet (to be included in HTML outputs).
-  --embed-css        Include the contents of the CSS in the HTML output.
-  --stdout           No output file is created and the result goes to stdout.
-  -h, --help         display help for command
+  -V, --version          output the version number
+  -p, --output <format>  Format of the output: md (default: "md")
```
Comment (rufuspollock) [inline README.md:35]: Surely `-o`

File: README.md [hunk 3]
```diff
@@ -10,47 +10,99 @@ Requires [NodeJS](https://nodejs.org/en/) 12+:
...
+  -h, --help             display help for command
+```
+
+#### Examples
```
Comment (rufuspollock) [inline README.md:39]: I'd move this up to level `##`

File: README.md [hunk 4]
```diff
@@ -10,47 +10,99 @@ Requires [NodeJS](https://nodejs.org/en/) 12+:
...
 
-To convert [`input.json`](test/fixtures/input.json) into [`output.md`](test/fixtures/output.md), we need to read the content of the input file pass it to the `toMarkDown` function. For example, the following function would return the contents of `output.md` when called with the contents of `input.json`:
-
-```javascript
-async function convert(path) {
-  const input = await fs.promises.readFile(path, "utf8");
-  return toMarkDown(input);
-}
-```
+## Usage
```
Comment (rufuspollock) [inline README.md:17]: Remove Usage at this point and move CLI and API up a level.

File: README.md [hunk 5]
```diff
@@ -10,47 +10,99 @@ Requires [NodeJS](https://nodejs.org/en/) 12+:
 
 ```console
 $ npm install
-$ npm link
 ```
```
Comment (rufuspollock) [inline README.md:14]: Might add a quick start with one example here (?)

**Review by rufuspollock — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cuducos) [reply README.md:30]: This is the text generated by the CLI and I tend to just copy & paste it. Commander wraps these lines, maybe. I tend to leave it as it is (after pasting) for a more realistic look & feel in the docs. What do you think?

**Pull request merged**